ganttproject/build.gradle

```diff
@@ -8,6 +8,9 @@ dependencies {
   compile fileTree(dir: project.ext.libDir, include: ['*.jar'])
   compile project(':..:biz.ganttproject.core')
   compile files('data/resources')
+  compile 'com.google.api-client:google-api-client:1.22.0'
```

Comment:
in fact we need to upgrade guava anyway, so you can remove `ganttproject-guava.jar`, take the new one and fix compiling errors if any